README.md

```diff
@@ -1019,6 +1020,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kunal Sangtiani](https://github.com/Kunals131)
   - [Kuwar Kapur](https://github.com/kuwarkapur)
 
+
```

Comment:
Hi @khalidx3 ,
Your name is now in alphabetical order, but you have add an extra new line at the end of the K section. Please remove it.